spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/SpannerReadConverter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.spanner.core.convert;
+
+import org.springframework.data.convert.CustomConversions;
+
+/**
+ * @author Balint Pato
+ */
+public class SpannerReadConverter extends AbstractSpannerCustomConverter {
+
+	public SpannerReadConverter(CustomConversions customConversions) {
+		super(customConversions, null);
+	}
+
+	@Override
+	public <T> T convert(Object sourceValue, Class<T> targetType) {
+		Class<?> sourceClass = sourceValue.getClass();
+		T result = null;
+		if (targetType.isAssignableFrom(sourceClass)) {
+			return (T) sourceValue;
+		}
+		else if (super.canConvert(sourceClass, targetType)) {
+			result = super.convert(sourceValue, targetType);
+		}
+		return result;
+	}
+
+	@Override
+	protected boolean canConvert(Class sourceType, Class targetType) {
+		return super.canConvert(sourceType, targetType);
```

Comment:
It looks like this is literally calling the parent version; is this overrided method necessary at all?